### PR TITLE
Fail on unknown prop

### PR DESCRIPTION
**Pull request by fenollp**

### CODE REVIEW

**Comment by ferd** [comment]
Can you give a quick description of how to at least reproduce the case you're fixing here so I can try and compare it?

File: src/rebar3_proper_prv.erl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 -module(rebar3_proper_prv).
+-behaviour(provider).
```
Comment (ferd) [inline src/rebar3_proper_prv.erl]: I didn't put that one here so I could compile and run stuff without depending on rebar3 being in the environment. It was a deliberate omission.

**Comment by fenollp** [comment]
Sure! Sorry about that.

These should fail with an helpful message:
1. `rebar3 proper -m this_module_does_not_exist`
1. `rebar3 proper -m this_module_does_exist` (but does not export any `prop_`s)
1. `rebar3 proper -m this_module_does_not_exist -p this_prop_doesnt_either`
1. `rebar3 proper -m this_module_does_not_exist -p this_prop_does_though_but_somewhere_else`
1. `rebar3 proper -p this_prop_is_in_no_module`

These shouldn't:
1. `rebar3 proper -p this_prop_exists_and_is_in_a_module_that_exists`
1. `rebar3 proper -p this_prop_exists_and_is_in_a_module_that_exists -m ie_this_module`

Backwards incompatibility:
- when there are no properties to check command will now fail, instead of happily showing "0/0 tests ran"

Will revert that behaviour line.

**Comment by ferd** [comment]
Merged and published to hex as 0.6.0. Thanks for the contribution!

**Pull request merged**